File: taller/AMarotoChivite-taller/AMarotoChivite-taller-Java/src/test/java/simulacionTaller/factories/ElectricianFactoryTest.java
package simulacionTaller.factories;

import org.junit.jupiter.api.Test;
import simulacionTaller.models.Electrician;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class ElectricianFactoryTest {

    @Test
    void createTest() {
        Electrician electrician = ElectricianFactory.create();
        // No nulo, para ver si se han asignado correctamente los datos
        assertNotNull(electrician);
    }
}

File: taller/AMarotoChivite-taller/AMarotoChivite-taller-kotlin/src/test/kotlin/simulacionTaller/factories/ElectricianFactoryTest.kt
package simulacionTaller.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class ElectricianFactoryTest {
    @Test
    fun createTest() {
        val electrician = ElectricianFactory.create()
        // No nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(electrician)
    }
}